app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java

```diff
@@ -92,14 +96,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         mSharedpreferences = PreferenceManager
                 .getDefaultSharedPreferences(getActivity());
         mSharedpreferences.registerOnSharedPreferenceChangeListener(this);
-        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
-        actionBar.hide();
+//        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
```

Comment:
remove comments

Reply:
okay. i am still working on issue number #622 as that contains 3-4 sub parts. sorry for the delay.